### PR TITLE
all python files are now formated with black

### LINKED ISSUE #22: migrate the whole python files to be formated by black

### PR DESCRIPTION
**Pull request by utnapischtim**
this close #22 

NOTE:
some configurations where necessary. flake8 line-length has to be set to 88
which is the default for black. but this was not enough some lines of black
where formated longer then 88 characters. found flake8-bugbear with B950.

with that and in combination with ignore=E501 it is possible to ignore long
lines, but if there are lines to long it will still point it out.

further also for isort some configuration was necessary

REFERENCES:
https://github.com/psf/black/blob/master/docs/compatible_configs.md#isort
https://github.com/psf/black/blob/master/docs/compatible_configs.md#flake8
https://github.com/PyCQA/flake8-bugbear#opinionated-warnings

### CODE REVIEW

File: invenio_config_tugraz/ext.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 """invenio module that adds tugraz configs."""
 
-from flask_babelex import gettext as _
+# from flask_babelex import gettext as _
```
Comment (mb-wali) [inline invenio_config_tugraz/ext.py:11]:
```suggestion

```

File: setup.py [hunk 2]
```diff
@@ -12,87 +12,86 @@
...
 
 install_requires = [
-    'Flask-BabelEx>=0.9.4',
-    'elasticsearch_dsl>=7.2.1',
-    'invenio-rdm-records~=0.18.3',
-    'invenio_search>=1.3.1',
-
+    "Flask-BabelEx>=0.9.4",
+    "elasticsearch_dsl>=7.2.1",
+    "invenio-rdm-records~=0.18.3",
```
Comment (mb-wali) [inline setup.py:41]:
```suggestion
    "invenio-rdm-records~=0.20.8",
```

**Review by mb-wali — CHANGES_REQUESTED** [review]
All good! See the comments - for little changes that requires.

File: setup.py (hunk 2 above)
Comment (utnapischtim) [reply setup.py:41]: in my opinion it is not the task of this PR to make this change. this pull request is about to move to black. this should be done in another pull request.

**Review by mb-wali — APPROVED** [review]

**Pull request merged**